akka-distributed-cluster-docs/src/main/paradox/guide/2-service-to-service.md

```diff
@@ -0,0 +1,58 @@
+# Part 2: Service to Service eventing
+
+To implement Service to Service eventing, we will use two services, the shopping cart defined in the previous step and 
```

Comment:
We might need some intro description to Service to Service eventing. What is the purpose of it and so on. Maybe that is in some of the other feature-summary pages?